core/templates/domain/community_dashboard/ReadOnlyFeaturedTranslationLanguageObjectFactory.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating and mutating instances of frontend
+ * featured translation language domain objects.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+export interface IFeaturedTranslationLanguageBackendDict {
```

Comment:
The "I" prefix is unnecessary for interfaces: it's enough to be describe the interface's intended behavior via the name.

Reply:
I believe all backend dicts have an I prefix currently - correct me if I'm wrong

Reply:
This pattern for prefixing interfaces with 'I' does seem common throughout the codebase, but TypeScript already expresses information in types, so generally names should not be decorated with information that is included in the type. Same reasoning for preferring using the `private` modifier over an underscore prefix.